css/style.css

```diff
@@ -637,20 +637,20 @@ ul {
   padding-right: 8px;
   font-size: 25px;
 }
-.col h3 {
-  font-size: 1.22rem;
+.col h3{
+  font-size: 1.1rem;
   /* font-size: 16px; */
   font-family: 'Montserrat', Arial, Helvetica, sans-serif;
   line-height: normal;
   margin-bottom: 5px;
-  font-weight: 600;
+  font-weight: 550;
   letter-spacing: 0.01rem;
 }
 .col p {
   font-family: 'Montserrat', Arial, Helvetica, sans-serif;
   font-size: 13px;
-  font-size: 0.87rem;
-  line-height: 1.6;
+  font-size: 0.8rem;
+  line-height: 1.5;
   letter-spacing: 0.099rem;
 }
 .col hr {

```

Comment:
What is the "col" stand for give meaningful name to all class

else everything is fantastic, good work @kartikayasija